### PR TITLE
Add faq

### PR DESCRIPTION
**Pull request by canyon289**
Happy to add questions too. Let me know if im missing any

### CODE REVIEW

File: _2017_pages/faq.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      <h2 class="first">Key Dates</h2>
+      <p>Async Conference Start: October 24th on PyMC Discourse</p>
+      <p>Beginner Zone: October 24th to October 7th on Zulip </p>
+      <p>Synchronous Conference: October 31st on Webex and Zulip</p>
+
+      <h2 class="first">Do I need to purchase a ticket?</h2>
+      <p> For the async beginner zone you <b>do not</b> need to purchase a ticket. These are
+        for the good of the community and open to anyone that would benefit.
+
+        A ticket is required for the synchronous conference.
```
Comment (OriolAbril) [inline _2017_pages/faq.html]: I would change this to something like:

```

<p>A ticket is only required for the synchronous conference.</p>

<p> To take part in any of the async activities such as the beginner zone or the talk discussion on Discourse you <b>do not</b> need to purchase a ticket.</p>

```

I removed the for the good of the community part too as I think both sync and async are for the good of the community

**Review by OriolAbril — COMMENTED** [review]
Other questions I have been asked:

* Will the presentations be posted publicly?
* Can I take part in both time zones?
* What questions will speakers answer during the live Q&A? (or similar question, how will questions be asked to speakers during the live Q&A?)
* Will live sessions be recorded and published?
* How can I get an invite to Zulip?

**Comment by canyon289** [comment]
Made all the edits. its a little disorganized but has the info. I can do reorganization in another PR

**Comment by canyon289** [comment]
Merging to get this out there and if we can make it better I will

**Pull request merged**